### DISCUSSION TITLE
Show Taskbar tray/icons on secondary monitors

### DISCUSSION THREAD

**Discussion by VelocityRa** [valinet/ExplorerPatcher#2525] [Ideas]
I have not come across any way of doing this besides using [DisplayFusion](https://www.displayfusion.com/) which is kind of performance-intensive, has some lacking features (it has its own taskbar implementation), and is paid software.

**Comment by pyrates999** [comment]
If it's performance intensive, then that is why MS didn't implement showing system tray icons on the taskbar of secondary monitors.
